Gemfile

```diff
@@ -9,6 +9,8 @@ gem 'rake'
 gem 'rspec', '~> 3.0'
 gem 'rspec-puppet', '~> 2.3'
 gem 'rspec-puppet-facts', '>= 1.7'
+gem 'beaker-rspec', {"groups"=>["system_tests"]}
```

Comment:
Let's get https://github.com/theforeman/foreman-installer-modulesync/pull/55 merged and then I'll base it on that.